s3transfer/manager.py

```diff
@@ -273,6 +293,46 @@ def _validate_all_known_args(self, actual, allowed):
                     "must be one of: %s" % (
                         kwarg, ', '.join(allowed)))
 
+    def _submit_transfer(self, call_args, submission_task_cls,
+                         extra_main_kwargs=None):
+        if not extra_main_kwargs:
+            extra_main_kwargs = {}
+
+        # Create a TransferFuture to return back to the user
+        transfer_future, components = get_transfer_future_with_components(
```

Comment:
Preferred way I like to handle this is via dep. injection if you want it to be separately tested.  I always find it a bit odd to have internal, encapsulated instance methods that essentially reach into the global namespace to leverage utility functions.  Sometimes it makes sense, but I find it easier if that dependency is made explicit in the `__init__`.